src/item/CarrotOnAStick.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+ */
+
+declare(strict_types=1);
+
+namespace pocketmine\item;
+
+class CarrotOnAStick extends Durable{
+
+	public function getMaxStackSize() : int{
+		return 1;
+	}
+
+	public function getMaxDurability() : int{
+		return 26;
```

Comment:
I find 25, is this normal ?

Reply:
https://minecraft.wiki/w/Carrot_on_a_Stick

In Java Edition this is 25 and in Bedrock it is 26.

Reply:
Thats not what this one says, need testing to be fixed ? https://minecraft.fandom.com/wiki/Carrot_on_a_Stick

Reply:
https://github.com/PowerNukkitX/PowerNukkitX/blob/c8164b1fd104151a6e88af9133e6f5209795e1bb/src/main/java/cn/nukkit/item/ItemTool.java#L50

PowerNukkitX is using 26 as the value, but if you want I can look with a resource pack at the value later

Reply:
Should be checked yes unless someone like another source?

Reply:
![image](https://github.com/pmmp/PocketMine-MP/assets/135014092/49c4d6b7-f98b-49f3-9c10-0b342a5313f6)

It is 26